src/pages/myGeotab/guides/concepts.tsx

```diff
@@ -0,0 +1,1265 @@
+        </p>
+        <p>
+            Taking the previous assumptions, what would the overhead be for making 1000 requests for road max speeds? When individual calls are made to the server for 1000 addresses; the base (minimum) HTTP and Network overhead is required for each of these calls. This would result in 80 seconds (80,000 milliseconds) of network overhead and 0.72 MB (750,000 bytes) in headers just going to and from the server. It can be clearly seen that a great deal of overhead can be generated by making small but repeated requests.
+        </p>
+        <p>
+            By using a MultiCall, the network and HTTP overhead remains at the cost of a single request. This brings the overhead back down to our original 80 milliseconds and 750 bytes. The server processes each request and returns an Array of results when complete.    
+        </p>
+        <p>
+            The above illustration is an extreme example to demonstrate the benefits of using a MultiCall. A MultiCall can (and should) be used to make short running calls of 2 or more requests more efficient than individual calls.    
+        </p>
+    </div>
+);
+
+const MultiCallBasicImplementation: ReactNode = (
+    <div className="paragraph" id="multiCallImplementation">
+        <p>
+            Making a MultiCall is simple, use the method “ExecuteMultiCall” with the parameter “calls” of JSON type Array. Each call should be formatted as an Object with property “method” of type string with the method name as its value and a property “params” of type Object with the method parameters as its properties. The parent “params” object will also need to contain the user credentials if they are required for at least one of the child methods being called. It is not necessary to include credentials with each child call.    
+        </p>
+        <CodeSample
+        language="javascript"
+        code={`{
+    "method": "ExecuteMultiCall",
+    "params": {
+        "calls": [
+            {
+                "method": "GetCountOf",
+                "params": {
+                    "typeName": "Device"
+                }
+            },
+            {
+                "method": "GetCountOf",
+                "params": {
+                    "typeName": "User"
+                }
+            }
+        ],
+        "credentials": {
+            "database": "demo",
+            "sessionId": "xxx",
+            "userName": "bob@geotab.com"
+        }
+    }
+}`
+}></CodeSample>
+        <p>Response:</p>
+        <CodeSample
+        language="javascript"
+        code={`{
+    "result": [
+        2340,
+        2022
+    ],
+    "jsonrpc": "2.0"
+}`
+}></CodeSample>
```

Comment:
```suggestion
}/>
```
This component can be self-closing since there are no children.